crates/re_space_view_time_series/src/space_view_class.rs

```diff
@@ -367,20 +366,20 @@ It can greatly improve performance (and readability) in such situations as it pr
 
         let auto_y = y_range.is_none();
 
-        // We don't want to allow scrolling when y is locked unless using the hoizontal scroll modifier.
-        // Otherwise the view "bounces" when we scroll and then reset to the locked range.
-        let allow_scroll = ui.input(|i| i.modifiers.contains(controls::HORIZONTAL_SCROLL_MODIFIER))
-            || !lock_y_during_zoom;
+        // We don't want to allow vertical when y is locked or else the view "bounces" when we scroll and
+        // then reset to the locked range.
+        if lock_y_during_zoom {
+            ui.input_mut(|i| i.smooth_scroll_delta.y = 0.0);
+        }
```

Comment:
doesn't that mean the scroll delta is zero everywhere?
hm, doesn't seem to affect anything else. okay